src/cmds/docs/index.js

```diff
@@ -119,7 +109,7 @@ exports.run = async function (opts) {
 
   const updatedDocs = await Promise.allSettled(
     files.map(async filename => {
-      const file = await readFile(path.join(folder, filename), 'utf8');
+      const file = await readFile(filename, 'utf8');
```

Comment:
Seems like this change is breaking the command functionality. I believe we need path.join(folder, filename) to read the file from the correct path.

Reply:
So I thought the same, but `files` are already being composed of fully pathed out files and this line was rewriting `path/to/doc.md` as `path/to/path/to/doc.md`.

Reply:
You're right that files is fully pathed out initially but we reassign it to allFiles later on. And we stripped out the full paths in allFiles. 

With these lines:

`
// Pull off the leading subdirectory, to keep things consistant with below
  const allFiles = readdirRecursive(folder).map(file => file.replace(`${folder}${path.sep}`, ''));
`
`
  // Strip out non-markdown files
  const files = allFiles.filter(file => file.endsWith('.md') || file.endsWith('.markdown'));
`

Reply:
So when we run updatedDocs, we don't have the fully pathed out filenames anymore and need to add back the folder lol

Reply:
Talked offline and it's failing when you pass a `../relative/dir` path to the `folder` arg. `./dir` paths work just fine. Working on a fix.

Reply:
Fixed in https://github.com/readmeio/rdme/pull/343/commits/c95aac8e3488e15b48919c8654edcf462b0c6592